.idea/runConfigurations/iam_coverage_report.xml

```diff
@@ -7,7 +7,7 @@
     <option name="INDEPENDENT_SCRIPT_WORKING_DIRECTORY" value="true" />
     <option name="SCRIPT_WORKING_DIRECTORY" value="$PROJECT_DIR$/api/pkg/apps/iam" />
     <option name="INDEPENDENT_INTERPRETER_PATH" value="true" />
-    <option name="INTERPRETER_PATH" value="/usr/bin/bash" />
+    <option name="INTERPRETER_PATH" value="/usr/local/bin/bash" />
```

Comment:
We keep getting changes to these files

Reply:
I'm not sure what to do about this one, it depends on the OS and I guess we use different ones...

Reply:
Nothing to do, but we need to fix it at a higher level